README.md

```diff
@@ -71,13 +71,42 @@ c.close()
 from confluent_kafka import avro
 from confluent_kafka.avro import AvroProducer
 
-value_schema = avro.load('ValueSchema.avsc')
-key_schema = avro.load('KeySchema.avsc')
+
+value_schema_str = """
+{
+   "namespace": "my.test",
+   "name": "value",
+   "type": "record",
+   "fields" : [
+     {
+       "name" : "name",
+       "type" : "string"
+     }
+   ]
+}
+"""
+
+key_schema_str = """
+{
+   "namespace": "my.test",
+   "name": "key",
+   "type": "record",
+   "fields" : [
+     {
+       "name" : "name",
+       "type" : "string"
+     }
+   ]
+}
+"""
+
+value_schema = avro.loads(value_schema_str)
+key_schema = avro.loads(key_schema_str)
 value = {"name": "Value"}
 key = {"name": "Key"}
 
-avroProducer = AvroProducer({'bootstrap.servers': 'mybroker,mybroker2', 'schema.registry.url': 'http://schem_registry_host:port'}, default_key_schema=key_schema, default_value_schema=value_schema)
-avroProducer.produce(topic='my_topic', value=value, key=key)
```

Comment:
why remove default_*_schema args in favour of per-produce-call schema args?

Reply:
The first thing I was looking to do was send a few messages on a topic.  But my topics have multiple messages per topic, so the default_*_schema didn't work for me.  I poked around a bit before finding the parameters to the produce call so wanted to capture it in the doc.  I'm not sure what the most common use case is but I thought passing the schemas on the produce call was the most flexible.

I'm happy to change this back if you think it makes more sense to use the default schemas.

Reply:
Sorry for the delay, I think you should change it back to using default schema, since that is what most people will likely be using.